packages/rsx/src/element.rs

```diff
@@ -130,38 +129,61 @@ impl Parse for Element {
                 continue;
             }
 
+            // Parse shorthand fields
+            if content.peek(Ident)
+                && !content.peek2(Brace)
+                && !content.peek2(Token![:])
+                && !content.peek2(Token![-])
+            {
+                let name = content.parse::<Ident>()?;
+                let name_ = name.clone();
+
+                // If the shorthand field is children, these are actually children!
+                if name == "children" {
+                    panic!("children must be wrapped in braces");
```

Comment:
This should probably return a syn error instead of panicing so that it highlights the right part of the rsx